mentat/terminal/client.py

```diff
@@ -115,6 +117,10 @@ def _init_signal_handlers(self):
     async def _startup(self):
         assert self.session is None, "TerminalClient already running"
 
+        # TODO: Right now, everything sent in Session.create is never printed because
+        # the PromptSessions and listening tasks haven't been set up yet; once we move
+        # the config to be client-side, we can get the config without setting up the
+        # session first, and we can move the session last
```

Comment:
I think we should update `broadcast.py` to handle this. A behavior that I think would be nice is having stream messages queued up when there's no listeners, then the second a new listener is "subscribed" the queue messages are copied over to the new subscriber queue which the subscriber (the Terminal Client in this case) can ingest and render.

Maybe make a github issue to pick this up later?

Reply:
Also like the queue idea